### PR TITLE
Moving Jill's story blog post into new blog

### PR DESCRIPTION
**Pull request by EllieNodder**
Feedback from the returners team asked us to add this old blog post onto the new blog page

### Trello card

https://trello.com/c/NClIOwaz/2669-content-changes-requested-by-returners-team

### CODE REVIEW

**Comment by peteryates** [comment]
I've rebased this and repushed - I would suggest not using a photo in this post though as the one we have is potato quality.

**Comment by gemmadallmandfe** [comment]
Changing to draft until @EllieNodder confirms that the age of this story with the returners' team

**Comment by EllieNodder** [comment]
> Changing to draft until @EllieNodder confirms that the age of this story with the returners' team

@gemmadallmandfe have asked Sharon from the returners' team but she's not sure so she's asked Christine. Not sure if there's anyone else I could ask to see if they know?

File: app/views/content/blog/jills-story-getting-back-into-the-classroom.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+title: Getting back into the classroom
+date: "2021-01-01"
+author: Jill
+images:
+  jill:
+    path: media/images/content/stories/stories-jill.jpg
+    thumbnail_path: media/images/content/stories/stories-jill.jpg
+    alt: Returning teacher Jill
+description: |-
+  Jill's route into teaching
```
Comment (gemmadallmandfe) [inline app/views/content/blog/jills-story-getting-back-into-the-classroom.md:11]: Need to have more of a description here as this appears on the blog landing page

File: app/views/content/blog/jills-story-getting-back-into-the-classroom.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+I continued to access professional development from DfE in behaviour management, and took a free teacher subject specialism training in maths, which was a unit from a master's course. I also volunteered two days per week for two terms, working with pupils with similar special educational needs, which allowed me to gain further experience.
+
+I was given the opportunity to teach maths to year nine pupils and the school very kindly provided me with a reference in addition to the one from the supply agency.
+
+My return to teaching adviser then arranged for me to attend Manchester Metropolitan University's (MMU) teacher engagement programme. Here, I accessed further DfE funded training in computing in the classroom, job application writing, interview tips, and much more.
+
+I was fortunate enough to land a job before the course even finished – and I would have been unlikely to be successful were it not for the training I had received as a result of my contact with the Return to Teaching Adviser service. MMU also worked to place other candidates in partner schools for work experience with the potential for a job offer.
```
Comment (gemmadallmandfe) [inline app/views/content/blog/jills-story-getting-back-into-the-classroom.md:35]: Suggest adding link to tta-service for 'return to teaching adviser service' (and also removing capitalisation)

**Comment by gemmadallmandfe** [comment]
@EllieNodder I've had a look and have a couple of minor suggestions

My other question is what we are planning to do with the My Story page on Jill? Will this be deleted - otherwise it is duplicate content? Plus this PR includes a redirect anyway

I know we need to look at the My Story section properly as a whole but need to deal with Jill's story either as part of this PR or in a separate one after this blog is published.

**Comment by EllieNodder** [comment]
> @EllieNodder I've had a look and have a couple of minor suggestions
> 
> My other question is what we are planning to do with the My Story page on Jill? Will this be deleted - otherwise it is duplicate content? Plus this PR includes a redirect anyway
> 
> I know we need to look at the My Story section properly as a whole but need to deal with Jill's story either as part of this PR or in a separate one after this blog is published.

Have pushed some edits based on your comments. And I think the plan was to delete Jill's story from the 'my story' section? Shall I start doing this now we're moving it over?

**Comment by peteryates** [comment]
I think we should just leave the story in place until we delete the section, happy to see this merged once people are happy with the contents.

**Comment by gemmadallmandfe** [comment]
@EllieNodder One final comment (sorry) - might be better to add a specific footer about returning to teaching rather than the generic one about becoming a teacher? The other returning blogs have some examples.

But let's get this merged tomorrow!

**Comment by EllieNodder** [comment]
> @EllieNodder One final comment (sorry) - might be better to add a specific footer about returning to teaching rather than the generic one about becoming a teacher? The other returning blogs have some examples.
> 
> But let's get this merged tomorrow!

@gemmadallmandfe @peteryates I've edited the footer, but when I've tried to go into the review app to check it, if I try and click on the 'returning to teaching' tag under the blog or page 3 of the blog, it says error 500, sorry, there's a problem with the website?

**Comment by peteryates** [comment]
@EllieNodder - the reason it failed is because the image wasn't registered in the new central list. If you look at my latest commit above you'll see how it works. I planned on giving you a intro to it but didn't find time. I'll set up a call.

**Comment by gemmadallmandfe** [comment]
@EllieNodder @peteryates This blog post is still giving a 500 error :-(

**Comment by peteryates** [comment]
@gemmadallmandfe - rebased and repushed, looks like it's working now. Looks like it was failing due to the previous build not deploying properly so the review app was a few versions behind.

**Review by gemmadallmandfe — APPROVED** [review]
Blog looks good to me

Can't test the redirect in dev though but happy to approve and for this to be merged

**Pull request merged**